package.json

```diff
@@ -56,35 +56,35 @@
     }
   },
   "devDependencies": {
-    "@nestjs/axios": "^1.0.1",
-    "@nestjs/cli": "^9.0.0",
-    "@nestjs/common": "^9.0.0",
```

Comment:
@jhaltercmu2020 you're just updating the dev dependencies here. I think what you are looking for is modifying the `peerDependencies`. You should probably do something like `"@nestjs/common": "^9.0.0 || ^10.0.0",` (etc.)

Additionally, you're upgrading the major versions of `jsonwebtoken` and `jwks-rsa`. Did you go through and make sure there aren't any breaking changes that need to be updated in the library?

Given that it doesn't look like any code needs to be changed to this library in order to support nestjs v10, I don't think this will be a major version release. However, I'd like you to test your PR on nest v9 and v10 just to be sure.

Reply:
@jspizziri I fixed this and tested it on both v9 and v10.